### DISCUSSION TITLE
Vue attach template ref to DialogPanel

### DISCUSSION THREAD

**Discussion by jiblett1000** [tailwindlabs/headlessui#1401] [Help]
Hello,

I'm looking to attach a template ref to the DialogPanel but it comes up null. I would rather not add a wrapper within if I can help it. I think it would be great if potentially all the components used defineExpose to expose the component root element to the parent so it can be used if/when needed. Perhaps this is already possible somehow?

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

This is a bug that will be fixed by #1404, if you have other issues like this, please open an issue with a reproduction repo / CodeSandbox attached.

**Reply by jiblett1000** [reply]
Thanks Robin! I appreciate your response. :) That is great to hear....wasn't entirely sure if that was already the intended functionality or not.
